docs/creating-a-package.md

```diff
@@ -123,7 +123,7 @@ like you.
 
 Style sheets for your package should be placed in the _stylesheets_ directory.
 Any style sheets in this directory will be loaded and attached to the DOM when
-your package is activated. Style sheets can be written as CSS or [LESS] (but
+your package is activated. Style sheets can be written as CSS or [LESS] \(but
```

Comment:
maybe make this `, but LESS is...` and drop the parens completely from that?

Seems like a bold statement like the recommended language can stand on its own.